include/swift/Basic/JSONSerialization.h

```diff
@@ -481,6 +481,16 @@ struct ScalarTraits<uint32_t> {
   static bool mustQuote(StringRef) { return false; }
 };
 
+#if defined(_MSC_VER)
+// In MSVC, 'unsigned long' is 32bit size and different from uint32_t,
+// and it is used to define swift::sys::ProcessId.
```

Comment:
ProcessId in swift defined as ```typedef llvm::sys::ProcessInfo::ProcessId ProcessId;```.
LLVM ProcessInfo coded.
```
struct ProcessInfo {
#if defined(LLVM_ON_UNIX)
  typedef pid_t ProcessId;
#elif defined(LLVM_ON_WIN32)
  typedef unsigned long ProcessId; // Must match the type of DWORD on Windows.
  .....
```
Changing ProcessId in swift means changing or ignoring LLVM definition.
I think reusing the LLVM definition is better.